spec/features/messaging_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.feature 'Messaging' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.